### PR TITLE
Translation for the page 'Installation -> Getting Started'

### LINKED ISSUE #39: Translation for the page 'Installation -> Getting Started'
# Please help us translating this page in Italian!
  
  **Aiutaci a tradurre questa pagina in Italiano!**

  You can find all the details about how to help here: https://github.com/reactjs/it.reactjs.org/issues/1
  
  ## GRAZIE! :smile:

### PR DESCRIPTION
**Pull request by deblasis**
Closes #39

### CODE REVIEW

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
-People come to React from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a practical approach, we hope you'll find this section helpful.
+Le persone che si avvicinano a React hanno background e stili di apprendimento differenti. A prescindere dal fatto che tu preferisca un approccio più teorico o pratico, speriamo che questa sezione ti sia di aiuto.
 
-* If you prefer to **learn by doing**, start with our [practical tutorial](/tutorial/tutorial.html).
-* If you prefer to **learn concepts step by step**, start with our [guide to main concepts](/docs/hello-world.html).
+* Se preferisci **imparare facendo**, inizia dal nostro [tutorial pratico](/tutorial/tutorial.html).
+* Se preferisci **imparare i concetti passo passo**, inizia dalla nostra [guida ai concetti principali](/docs/hello-world.html).
 
-Like any unfamiliar technology, React does have a learning curve. With practice and some patience, you *will* get the hang of it.
+Come ogni nuova tecnologia, anche React ha una curva di apprendimento. Con la pratica ed un po' di pazienza, lo imparerai fino in fondo, *garantito*.
```
Comment (LucaBlackDragon) [inline content/docs/getting-started.md]: 👍

File: content/docs/getting-started.md [hunk 2]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
 
-### Practical Tutorial {#practical-tutorial}
+### Tutorial Pratico {#practical-tutorial}
 
-If you prefer to **learn by doing,** check out our [practical tutorial](/tutorial/tutorial.html). In this tutorial, we build a tic-tac-toe game in React. You might be tempted to skip it because you're not building games -- but give it a chance. The techniques you'll learn in the tutorial are fundamental to building *any* React apps, and mastering it will give you a much deeper understanding.
+Se preferisci **imparare facendo,** dà uno sguardo al nostro [tutorial pratico](/tutorial/tutorial.html). In esso, sviluppiamo un gioco tic-tac-toe (tris) in React. Potresti avere la tentazione di non seguire il tutorial perchè normalmente non sviluppi giochi -- ma dagli una possibilità. Le tecniche che imparerai nel tutorial sono fondamentali alla costruzione di *ogni* app in React, impararle ti darà una comprensione molto più profonda.
```
Comment (LucaBlackDragon) [inline content/docs/getting-started.md]:
```suggestion
Se preferisci **imparare facendo,** dà uno sguardo al nostro [tutorial pratico](/tutorial/tutorial.html). In esso, sviluppiamo un gioco tic-tac-toe (tris) in React. Potresti avere la tentazione di non seguire il tutorial perchè normalmente non sviluppi giochi -- ma dagli una possibilità. Le tecniche che imparerai nel tutorial sono fondamentali per la costruzione di *ogni* app in React, impararle ti darà una comprensione molto più profonda.
```

File: content/docs/getting-started.md [hunk 3]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
+Il [glossario](/docs/glossary.html) contiene una panoramica dei temini più comuni che troverai nella documentazione di React. Esiste anche una sezione con le FAQ (domande frequenti) dove potrai trovare brevi domande e risposte riguardo i temi più comuni, inclusi [creare richieste AJAX](/docs/faq-ajax.html), [stato dei componenti](/docs/faq-state.html) e [struttura dei file](/docs/faq-structure.html).
 
-## Staying Informed {#staying-informed}
+## Rimanere Aggiornati {#staying-informed}
 
-The [React blog](/blog/) is the official source for the updates from the React team. Anything important, including release notes or deprecation notices, will be posted there first.
+Il [blog di React](/blog/) è la fonte ufficiale per gli aggiornamenti dal team di React. Ogni annuncio importante, incluse note di rilascio o di obsolescenza, verranno postate qui inizialmente.
 
-You can also follow the [@reactjs account](https://twitter.com/reactjs) on Twitter, but you won't miss anything essential if you only read the blog.
+Puoi inoltre seguire l'[account @reactjs](https://twitter.com/reactjs) su Twitter, non ti perderai nulla di essenziale leggendo solo il blog.
```
Comment (LucaBlackDragon) [inline content/docs/getting-started.md]:
```suggestion
Puoi inoltre seguire l'[account @reactjs](https://twitter.com/reactjs) su Twitter, ma non ti perderai nulla di essenziale leggendo solo il blog.
```

File: content/docs/getting-started.md [hunk 4]
```diff
@@ -18,109 +18,109 @@ redirect_from:
...
+Questa documentazione riflette sempre l'ultima versione stabile di React. A partire da React 16, puoi trovare le vecchie versioni della documentazione [in una pagina separata](/versions). Nota che la documentazione per le versioni precedenti viene salvata al momento del rilascio, non riceve più aggiornamenti in seguito.
 
-## Something Missing? {#something-missing}
+## Manca Qualcosa? {#something-missing}
 
-If something is missing in the documentation or if you found some part confusing, please [file an issue for the documentation repository](https://github.com/reactjs/reactjs.org/issues/new) with your suggestions for improvement, or tweet at the [@reactjs account](https://twitter.com/reactjs). We love hearing from you!
+Se ritieni che la documentazione abbia delle lacune, oppure se trovi che alcune parti non siano chiare, per favore [crea una issue nella repository della documentazione ](https://github.com/reactjs/it.reactjs.org/issues/new) con i tuoi suggerimenti, oppure invia un tweet all'account [@reactjs](https://twitter.com/reactjs). Adoriamo ricevere feedback!
```
Comment (LucaBlackDragon) [inline content/docs/getting-started.md]:
```suggestion
Se ritieni che la documentazione abbia delle lacune, oppure se trovi che alcune parti non siano chiare, per favore [crea una issue nel repository della documentazione ](https://github.com/reactjs/it.reactjs.org/issues/new) con i tuoi suggerimenti, oppure invia un tweet all'account [@reactjs](https://twitter.com/reactjs). Adoriamo ricevere feedback!
```
I've treated "repository" as a male noun (as in [https://it.wikipedia.org/wiki/Repository](https://it.wikipedia.org/wiki/Repository))

File: static/html/single-file-example.html [hunk 5]
```diff
@@ -2,35 +2,35 @@
...
+        <h1>Ciao Mondo</h1>,
         document.getElementById('root')
       );
 
     </script>
     <!--
-      Note: this page is a great way to try React but it's not suitable for production.
-      It slowly compiles JSX with Babel in the browser and uses a large development build of React.
+      Nota: questa pagina è un ottimo modo per provare React ma non è adatta all'uso in produzione.
+      Essa compila il codice JSC con Babel nel browser in modo molto lento ed utilizza una build di sviluppo di React non compressa.
```
Comment (LucaBlackDragon) [inline static/html/single-file-example.html]:
```suggestion
      Essa compila il codice JSX con Babel nel browser in modo molto lento ed utilizza una build di sviluppo di React non compressa.
```

**Comment by deblasis** [comment]
@LucaBlackDragon 😄 awesome!

Wow, I always treated *repository* as a female noun! You shocked me haha! 🤣

**Comment by LucaBlackDragon** [comment]
> 
> 
> @LucaBlackDragon 😄 awesome!
> 
> Wow, I always treated _repository_ as a female noun! You shocked me haha! 🤣

Actually I always wonder if *I* am mistaken on these matters... hadn't I checked Wikipedia I'd have left that as it was!

**Pull request merged**